From d0c1759bb2e45c649b0c323054973443e13b6cc4 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <jan.steffens@gmail.com>
Date: Tue, 1 Oct 2019 14:06:10 +0000
Subject: [PATCH 2/4] config: Improve config lookup error messages

Match the message for wifi/submit-data and add some quotes.

Downgrade the message for wifi/url from a warning to a debug message.
---
 src/gclue-config.c | 11 +++++++----
 1 file changed, 7 insertions(+), 4 deletions(-)

diff --git a/src/gclue-config.c b/src/gclue-config.c
index df4adcc..00dd52e 100644
--- a/src/gclue-config.c
+++ b/src/gclue-config.c
@@ -232,7 +232,8 @@ load_wifi_config (GClueConfig *config)
                                                 "url",
                                                 &error);
         if (error != NULL) {
-                g_warning ("%s", error->message);
+                g_debug ("Failed to get config \"wifi/url\": %s",
+                         error->message);
                 g_clear_error (&error);
                 priv->wifi_url = g_strdup (DEFAULT_WIFI_URL);
         }
@@ -242,7 +243,7 @@ load_wifi_config (GClueConfig *config)
                                                     "submit-data",
                                                     &error);
         if (error != NULL) {
-                g_debug ("Failed to get config wifi/submit-data: %s",
+                g_debug ("Failed to get config \"wifi/submit-data\": %s",
                          error->message);
                 g_error_free (error);
 
@@ -254,7 +255,8 @@ load_wifi_config (GClueConfig *config)
                                                        "submission-url",
                                                        &error);
         if (error != NULL) {
-                g_debug ("No wifi submission URL: %s", error->message);
+                g_debug ("Failed to get config \"wifi/submission-url\": %s",
+                         error->message);
                 g_error_free (error);
                 priv->wifi_submit_url = g_strdup (DEFAULT_WIFI_SUBMIT_URL);
         }
@@ -264,7 +266,8 @@ load_wifi_config (GClueConfig *config)
                                                         "submission-nick",
                                                         &error);
         if (error != NULL) {
-                g_debug ("No wifi submission nick: %s", error->message);
+                g_debug ("Failed to get config \"wifi/submission-nick\": %s",
+                         error->message);
                 g_error_free (error);
         }
 }
-- 
2.23.0

